src/main/webapp/WEB-INF/views/admin/allProducts.jsp

```diff
@@ -0,0 +1,34 @@
+<%@ taglib prefix="c" uri="http://java.sun.com/jsp/jstl/core" %>
+<%@ page contentType="text/html;charset=UTF-8" language="java" %>
+<html>
+<head>
+    <title>All products</title>
+</head>
+<body>
+<h1>All products page</h1>
+<table border="1">
+    <tr>
+        <th>ID</th>
+        <th>Name</th>
+        <th>Price</th>
+    </tr>
+    <c:forEach var="product" items="${products}">
+        <tr>
+            <td>
+                <c:out value="${product.id}"/>
+            </td>
+            <td>
+                <c:out value="${product.name}"/>
+            </td>
+            <td>
+                <c:out value="${product.price}"/>
+            </td>
+            <td>
+                <a href="${pageContext.request.contextPath}/admin/product/delete?id=${product.id}">Delete</a>
+            </td>
+        </tr>
+    </c:forEach>
+</table>
+<a href="${pageContext.request.contextPath}">Go to the main page</a>
```

Comment:
let's also have a link to the form that adds a new product to the storage

Reply:
done